### PR TITLE
bump version of nbconvert

### PR DESCRIPTION
**Pull request by rkpattnaik780**
Bump version of `nbconvert` to fix the CVE issues introduced by `mistune@2.0.3`.

### How was this pull request tested?

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by lresende** [comment]
@rkpattnaik780 could you please sign your commit?

**Comment by rkpattnaik780** [comment]
> @rkpattnaik780 could you please sign your commit?

Done. The checks are also passing now!

**Comment by shalberd** [comment]
@lresende this also fixes https://github.com/elyra-ai/elyra/issues/3228

File: pyproject.toml [hunk 1]
```diff
@@ -32,7 +32,7 @@ dependencies = [
     "MarkupSafe>=2.1",
     "minio>=7.0.0,!=7.2.1",
     "nbclient>=0.5.1",
-    "nbconvert>=6.5.1",
+    "nbconvert==7.1.0",
```
Comment (shalberd) [inline pyproject.toml:35]: fine, yes.
Should also be added to https://github.com/elyra-ai/elyra/blob/main/etc/generic/requirements-elyra.txt#L11

Comment (lresende) [reply pyproject.toml:35]: Please also add there

Comment (shalberd) [reply pyproject.toml:35]: @harshad16 I basically said, it absolutely makes sense to update the version of nbconvert in two places simultaneously.

**Pull request closed without merging**